docs/pr-guide.md

```diff
@@ -0,0 +1,66 @@
+3. Create a folder `site/content/protocols/<protocol-name>/<protocol-version>`
+4. Create a protocol definition document in that folder as a markdown file named `readme.md`. Use the following example as a [reference](./protocol-example.md)
+5. Send a Pull Request to the main branch
+
+## Update an existing protocol definition
+1. Fork repo
+2. Decide if the update is
+    - a fix
+    - compatible change
+    - non-compatible or breaking change
+
+3. If update is a `fix`
+    * the version must be the same
+    * make changes in existing `site/content/protocols/<protocol-name>/<protocol-version>/readme.md`
+    * send a PR to the main branch
+    * add `[fix]` postfix to the PR title
+4. If update is a `compatible change`
+    - define a new version where minor part must be incremented (for example, from 0.1 to 0.2)
+    - copy the previous readme.md to the new folder `site/content/protocols/<protocol-name>/<new-protocol-version>`
+    - make changes in the `readme.md`
+    - send a PR to the main branch
+    - add `[compatible]` postfix to the PR title
+5. If update is a `non-compatible change`
+    - define a new version where major part must be incremented (for example, from 0.1 to 1.0)
+    - copy the previous `readme.md` to the new folder `site/content/protocols/<protocol-name>/<new-protocol-version>`
+    - make changes in the `readme.md`
+    - send a Pull Rrequest to the main branch
+    - add `[non-compatible]` postfix to the PR title
+
+
+## Details
+#### Protocol Name
+`<protocol-name>` folder — It's the name of your protocol. It follows SLUG pattern and may consist of only lower cased latin letters, digits and `-` as separator e.g `my-cool-protocol`. Must be unique. For example, `my-protocol` and `myprotocol` are the same.
+
+
+#### Protocol Version
+`<protocol-version>` folder — follows [semver](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0003-protocols/README.md#semver-rules-for-protocols) without patch version. Two digits (major and minor version) separated by `.` e.g `1.0`.
+
+
+#### Metadata
+Placed at the top of the protocol definition between `---`.
+Uses [yaml](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html) syntax.
+The following fields must be specified:
+  - `title` — the official name of the protocol. Should be the same as `<protocol-name>` folder but user friendy. Will be displayed in search results and at protocol details page. 
+  - `piuri` — DIDComm protocols are uniquely identified by a special URI called a PIURI `https://didcomm.org/<protocol-name>/<version>`. e.g `https://didcomm.org/my-cool-protocol/1.0`
+  - `publisher` — the GitHub username person, who publish the protocol
+  - `licence` — protocol license
+  - `tags` — some hashtags that might be used to associate the protocol with topics. It's recommended to re-use existing tags before creating a new similar one (for example, consider re-using existing `finance` tag instead of creating a new one `financial`)
```

Comment:
```suggestion
  - `tags` — some hashtags that might be used to associate the protocol with topics. It's recommended to re-use existing tags before creating new similar ones (for example, consider re-using existing `finance` tag instead of creating a new one called `financial`).
```